vrc-get-gui/app/projects/manage/page.tsx

```diff
@@ -1376,17 +1400,22 @@ const PackageVersionSelector = memo(function PackageVersionSelector(
 	const incompatibleNames = [...pkg.unityIncompatible.keys()];
 
 	return (
-		<VGSelect value={<PackageInstalledInfo pkg={pkg}/>}
-							className={`border-blue-gray-200 ${pkg.installed?.yanked ? "text-red-700" : ""}`}
-							onChange={onChange}
-							disabled={locked}
-		>
-			{versionNames.map(v => <VGOption key={v} value={v}>{v}</VGOption>)}
-			{(incompatibleNames.length > 0 && versionNames.length > 0) && <hr className="my-2"/>}
-			{incompatibleNames.length > 0 &&
-				<Typography className={"text-sm"}>{tc("projects:manage:incompatible packages")}</Typography>}
-			{incompatibleNames.map(v => <VGOption key={v} value={v}>{v}</VGOption>)}
-		</VGSelect>
+		<Select onValueChange={onChange} disabled={locked}>
+			<SelectTrigger>
+				<SelectValue
+						placeholder={<PackageInstalledInfo pkg={pkg}/>}
+						className={`border-primary/10 ${pkg.installed?.yanked ? "text-destructive" : ""}`} />
```

Comment:
https://www.radix-ui.com/primitives/docs/components/select#value

> The part that reflects the selected value. By default the selected item's text will be rendered. if you require more control, you can instead control the select and pass your own children. 

it might be better to pass PackageInstalledInfo as children? 

I think this  may fix the problem noticed by @Sayamame-beans https://github.com/vrc-get/vrc-get/pull/1028#issuecomment-2131319312

Reply:
hm that approach doesn't seem to work.

![image](https://github.com/vrc-get/vrc-get/assets/54484875/866983ab-2bf2-4af8-b3a8-d6ed3983e09f)

I've noticed another bug that goes along with this issue that affects all `<Select>` components. I believe it's best I play around with the [value prop](https://www.radix-ui.com/primitives/docs/components/select#root) for `<Select>` until I get somewhere as the docs say it should be used in conjunction with `onValueChange`.

| What it looks like by default |
| --- |
| ![image](https://github.com/vrc-get/vrc-get/assets/54484875/725019ff-fcc9-4a4b-8bf3-1a73651ba7c1) |

| What it should look like |
| --- |
| ![image](https://github.com/vrc-get/vrc-get/assets/54484875/4271f0d3-9071-4be4-824d-bfd9f9d33fc2) |

When in the project manage page, if the checkmark is already there meaning the value is already selected, trying to select it again won't open the dialog.

![image](https://github.com/vrc-get/vrc-get/assets/54484875/bf1d9d60-6b53-4300-8503-97963643abf1)

I'll see what I can come up with soon.